lib/jsdom/living/nodes/HTMLFormElement.webidl

```diff
@@ -1,24 +1,25 @@
-[OverrideBuiltins]
+[Exposed=Window,
+ OverrideBuiltins,
+ LegacyUnenumerableNamedProperties,
```

Comment:
Given that we haven't added the named/indexed properties for the form element, I'm not sure we should add these.